CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.10.3
```

Comment:
Question: should this be a breaking change since we require a newer `protobuf` version?

If someone just does a `pub global activate` and get the latest without updating `protobuf`, they will generate broken code.

Reply:
I would say so.